health-services/project/src/main/resources/db/migration/main/V20220223043100__project_clientrefid_ddl.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ADDRESS ADD COLUMN IF NOT EXISTS clientReferenceId character varying(64) UNIQUE;
```

Comment:
This is already present in the previous migration file.

Reply:
we have added `IF NOT EXISTS` because deployments can run in any order.